app/ui-react/packages/ui/src/Integration/Details/IntegrationDetailInfo.tsx

```diff
@@ -24,29 +25,32 @@ export class IntegrationDetailInfo extends React.PureComponent<
   public render() {
     return (
       <div className="integration-detail-info">
-        {this.props.name}
-        <>
-          &nbsp;&nbsp;&nbsp;&nbsp;
-          {this.props.currentState === 'Pending' && (
-            <IntegrationStatusDetail
-              targetState={this.props.targetState}
-              value={this.props.monitoringValue}
-              currentStep={this.props.monitoringCurrentStep}
-              totalSteps={this.props.monitoringTotalSteps}
-              logUrl={this.props.monitoringLogUrl}
-              i18nProgressPending={this.props.i18nProgressPending}
-              i18nProgressStarting={this.props.i18nProgressStarting}
-              i18nProgressStopping={this.props.i18nProgressStopping}
-              i18nLogUrlText={this.props.i18nLogUrlText}
-            />
-          )}
-          {this.props.currentState === 'Published' && this.props.version && (
-            <>
-              <span className="pficon pficon-ok" />
-              &nbsp;Published version {this.props.version}
-            </>
-          )}
-        </>
+        <div className="integration-detail-info__title">
+          {this.props.name}
+          <>
+            &nbsp;&nbsp;&nbsp;&nbsp;
+            {this.props.currentState === 'Pending' && (
+              <IntegrationStatusDetail
+                targetState={this.props.targetState}
+                value={this.props.monitoringValue}
+                currentStep={this.props.monitoringCurrentStep}
+                totalSteps={this.props.monitoringTotalSteps}
+                logUrl={this.props.monitoringLogUrl}
+                i18nProgressPending={this.props.i18nProgressPending}
+                i18nProgressStarting={this.props.i18nProgressStarting}
+                i18nProgressStopping={this.props.i18nProgressStopping}
+                i18nLogUrlText={this.props.i18nLogUrlText}
+              />
+            )}
+            {this.props.currentState === 'Published' && this.props.version && (
+              <>
+                <span className="pficon pficon-ok" />
+                &nbsp;Published version {this.props.version}
+              </>
+            )}
+          </>
+        </div>
+        {this.props.status}
```

Comment:
This doesn't really look good when the integration is running:

![image](https://user-images.githubusercontent.com/351660/58354384-43614b80-7e3f-11e9-8e4a-99de341f4933.png)

We don't show this status in the angular app:

![image](https://user-images.githubusercontent.com/351660/58354440-71469000-7e3f-11e9-9be9-a3d51f02ca60.png)

Reply:
This was here already:

![image](https://user-images.githubusercontent.com/351660/58354521-b5d22b80-7e3f-11e9-80ed-1d9422941b77.png)

This is the missing bit:

![image](https://user-images.githubusercontent.com/351660/58354543-cbdfec00-7e3f-11e9-9493-31b0414a65f9.png)